ambry-api/src/main/java/com.github.ambry/config/FrontendConfig.java

```diff
@@ -160,6 +160,17 @@
   @Default("28 * 24 * 60 * 60")
   public final long chunkUploadInitialChunkTtlSecs;
 
+  public static final String FAIL_TTL_REQUIRED_NOT_PROVIDED_KEY = PREFIX + "fail.ttl.required.not.provided";
+  @Config(FAIL_TTL_REQUIRED_NOT_PROVIDED_KEY)
+  @Default("false")
+  public final boolean failTtlRequiredNotProvided;
+
+  public static final String MAX_ACCEPTABLE_TTL_SECS_IF_TTL_REQUIRED_KEY =
+      PREFIX + "max.acceptable.ttl.secs.if.ttl.required";
+  @Config(MAX_ACCEPTABLE_TTL_SECS_IF_TTL_REQUIRED_KEY)
+  @Default("30 * 24 * 60 * 60")
```

Comment:
Does this actually resolve the math expression inside the quotes?

Reply:
No -  just for display. The actual code is in the constructor (`TimeUnit.DAYS.toSeconds(30)`)